README.Rmd

```diff
@@ -40,78 +41,149 @@ devtools::install_github("ropensci/mregions")
 
 ```{r}
 library("mregions")
+library("sf")
+
+# helper library
+library("leaflet")
 ```
 
+
 ## GeoJSON
 
-Get region
+### Marine Ecoregions of the World (MEOW)
 
-```{r}
-res1 <- mr_geojson(key = "Morocco:dam")
+Get Data
+
+```{r, eval=FALSE}
+ecoregions_geoJSON <- mr_geojson(key = "Ecoregions:ecoregions")
+# check number of features
+length(ecoregions_geoJSON$features)
 ```
 
-Get helper library
+Plot Data
 
 ```{r eval=FALSE}
-install.packages("leaflet")
+leaflet() %>%
+  addProviderTiles(provider = 'OpenStreetMap') %>%
+  addGeoJSON(geojson = ecoregions_geoJSON$features) %>%
+  fitBounds(-160,-52,160,60)
 ```
+![map1](tools/img/leaf1_ecoregions.png)
+
 
-Plot data
+
+### Maritime Boundaries (EEZ)
+
+Get Data
+
+```{r}
+eezboundaries_geoJSON <- mr_geojson(key = "MarineRegions:eez_boundaries")
+```
+
+Plot Data
 
 ```{r eval=FALSE}
-library('leaflet')
 leaflet() %>%
   addProviderTiles(provider = 'OpenStreetMap') %>%
-  addGeoJSON(geojson = res1$features) %>%
-  setView(-3.9, 35, zoom = 10)
+  addGeoJSON(geojson = eezboundaries_geoJSON$features) %>%
+  fitBounds(-160,-52,160,60)
 ```
-
-![map](tools/img/leaf1.png)
+![map2](tools/img/leaf1_eezboundaries.png)
 
 ## Shape
 
+### Marine Ecoregions of the World (MEOW)
+
 Get region
 
 ```{r}
-res2 <- mr_shp(key = "MarineRegions:eez_iho_union_v2", maxFeatures = 5)
+ecoregions_shp <- mr_shp(key = "Ecoregions:ecoregions", maxFeatures = 50)
+# check class
+class(ecoregions_shp)
 ```
 
-Get helper library
+Plot data
 
 ```{r eval=FALSE}
-install.packages("leaflet")
+leaflet() %>%
+  addProviderTiles(provider = 'OpenStreetMap') %>%
+  addPolygons(data = ecoregions_shp)
+```
+
+![map4](tools/img/leaf2_ecoregions.png)
+### Maritime Boundaries (EEZ)
+
+Get region
+
+```{r}
+eezboundaries_shp <- mr_shp(key = "MarineRegions:eez_boundaries", maxFeatures = 50)
 ```
 
 Plot data
 
 ```{r eval=FALSE}
-library('leaflet')
 leaflet() %>%
   addProviderTiles(provider = 'OpenStreetMap') %>%
-  addPolygons(data = res2)
+  addPolygons(data = eezboundaries_shp)
 ```
 
-![map2](tools/img/leaf2.png)
+![map5](tools/img/leaf2_eezboundaries.png)
 
 ## Convert to WKT
 
-From geojson
+### Marine Ecoregions of the World (MEOW)
 
-```{r eval=FALSE}
-res3 <- mr_geojson(key = "Morocco:dam")
-mr_as_wkt(res3, fmt = 5)
+From GeoJSON
 
-#> [1] "MULTIPOLYGON (((41.573732 -1.659444, 45.891882 ... cutoff
+```{r eval=FALSE}
```

Comment:
Why not `eval = TRUE` and print the object?
e.g. replace `class(ecoregions_wkt1)` by directly `ecoregions_wkt1`